src/document.rs

```diff
@@ -13,6 +13,7 @@ use rss;
 
 use cobalt_model;
 use cobalt_model::files;
+use cobalt_model::permalink::{explode_permalink, format_url_as_file};
```

Comment:
Could you switch to `use cobalt_model::permalink`?

First, I prefer that for the sake of making it easier to see where the code is coming from.

Second, this masked the fact that certain tests weren't moved over (github doesn't let me comment on code outside of the diff)

Reply:
Oh yeah, sorry I didn't noticed that it was not the adopted coding style, correcting this tomorrow :D (bed time here ^^')

Reply:
I wasn't going to care enough but it caused us to miss the tests.